protobuf-kafkaconnect-converter/src/main/java/com/amazonaws/services/schemaregistry/kafkaconnect/protobuf/fromconnectschema/FieldBuilder.java

```diff
@@ -0,0 +1,265 @@
+package com.amazonaws.services.schemaregistry.kafkaconnect.protobuf.fromconnectschema;
```

Comment:
Please include the copyright information in all the files.

Reply:
Added